src/components/crud/delete.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable  import/no-cycle, import/no-mutable-exports, import/prefer-default-export */
+import { tasks } from '../../index.js';
+import { renderAll } from '../render.js';
+import { storeToLocal } from '../storage.js';
+import { updateIndex } from '../supportive.js';
+
+const deleteTask = (event) => {
+  if (event.target.classList.contains('delete-btn')) {
+    const { index } = event.target.dataset;
+    tasks.splice(index, 1);
+    // Re-render the task list
```

Comment:
Kindly remove line number 1 and 11 in order to make your code cleaner and implement javascript best practice [YAGNI](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/lesson_DRY_KISS_YAGNI.md).